src/main/java/org/assertj/core/api/AbstractAssert.java

```diff
@@ -83,9 +83,7 @@ protected WritableAssertionInfo getWritableAssertionInfo() {
    * defined by the user with {@link #overridingErrorMessage(String, Object...)}.
    * <p>
    * Example :
-   * 
-   * <pre><code class='java'>
-   * public TolkienCharacterAssert hasName(String name) {
+   * <pre><code class='java'> public TolkienCharacterAssert hasName(String name) {
```

Comment:
No because the next lines start also with a space, they should have the same indentation.